webview/src/experiments/components/table/RowContextMenu.tsx

```diff
@@ -175,7 +175,7 @@ const getSingleSelectMenuOptions = (
 
   const notExperiment = depth !== 1
 
-  const withId = (
+  const notRunningWithId = (
```

Comment:
I have now been caught out by this POS a few times. It silently added `|| hasRunningExperiment` to the hidden options which leads to plenty of head scratching. Name is updated to reflect what it is actually doing.